cluster-autoscaler/hack/update-faq-flags.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+###
+# This script get flags from cluster-autoscaler and generate new flags table
+###
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_ROOT=$(realpath $(dirname "${BASH_SOURCE[0]}"))/..
+AUTOSCALER="${SCRIPT_ROOT}/cluster-autoscaler-$(go env GOARCH)"
+TARGET_FILE="${SCRIPT_ROOT}/FAQ.md"
+
+pushd "${SCRIPT_ROOT}" >/dev/null
+[ -e "${AUTOSCALER}" ] || make build
```

Comment:
Minor nit for a follow-up that non Linux users may need to tweak their setups due to the Makefile defaulting to `GOOS=Linux`.